sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/SendSmsToGroup.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.sms.samples.quickstart;
+
+import com.azure.communication.sms.SmsClient;
+import com.azure.communication.sms.SmsClientBuilder;
+import com.azure.communication.sms.models.SmsSendOptions;
+import com.azure.communication.sms.models.SmsSendResult;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Context;
+
+import java.util.Arrays;
+
+public class SendSmsToGroup {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
```

Comment:
With samples, we usually put a fake value. E.g. 
` String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
  String accessKey = "SECRET";`

Reply:
to be in sync with other sdks we what to make this a runable example

Reply:
But users will need to perform a separate step to set up the system environment variables anyway, so it will not be runnable as is right?